mysite/code/services/AddonUpdater.php

```diff
@@ -227,6 +238,8 @@ private function updateVersion(Addon $addon, Version $package)
         $version->License = $package->getLicense();
         // $version->Support = $package->getSupport();
 
+        echo " - Processed version {$version->Version}\n";
```

Comment:
This is a side note: I'd really love to see an output interface used instead of direct echos. Related https://github.com/silverstripe/silverstripe-framework/issues/7509 and https://github.com/silverstripe/silverstripe-framework/issues/6524

Reply:
I agree but I think that would be an API breakage, so I don't think it's part of this PR?

Reply:
Totally =)